bundle/manifests/oadp-operator.clusterserviceversion.yaml

```
@@ -809,6 +809,8 @@ spec:
                   value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
                 - name: RELATED_IMAGE_DATA_MOVER_DUMMY_POD
                   value: quay.io/konveyor/rsync-transfer:latest
+                - name: RELATED_IMAGE_MUSTGATHER_RHEL8
```

```suggestion
                - name: RELATED_IMAGE_MUSTGATHER
```
must it include an EL designation?